plugin.js

```diff
@@ -16,13 +16,15 @@ const defaultOptions = {
 function verifyBearerAuthFactory (options) {
   const _options = Object.assign({}, defaultOptions, options)
   if (_options.keys instanceof Set) _options.keys = Array.from(_options.keys)
-  const { keys, errorResponse, contentType, bearerType, auth, addHook = true } = _options
-
+  const { keys, errorResponse, contentType, bearerType, auth, addHook = true, verifyErrorLogLevel = 'error' } = _options
+  const inValidLogLevelError = Error('invalid log level')
+  if (verifyErrorLogLevel == null || ['debug', 'info', 'warn', 'error'].indexOf(verifyErrorLogLevel) < 0) throw inValidLogLevelError
```

Comment:
Nits:
I would use includes() instead of indexOf() as it seems to be faster
e.g.: https://www.measurethat.net/Benchmarks/ShowResult/259746